public_html/app/controllers/Users.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+    class Users extends Controller {
+        public function __construct() {
+            $this->userModel = $this->model('User');
+            $this->validator = new Validator();
+        }
+
+        // REGISTER
+        public function register() {
+
+            // IF POST METHOD
+            if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+                // Sanitize POST data
+                $_POST = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+                // Init data
+                $data = [
+                    'name' => trim($_POST['name']),
+                    'email' => trim($_POST['email']),
+                    'password' => trim($_POST['password']),
+                    'confirm_password' => trim($_POST['confirm_password'])
+                ];
+                // Validations
+                $validateData = [
```

Comment:
do not use rules in function, move it to constant or create function for this.
for example - methodsValidation() .